apps/provisioning_api/lib/Controller/AUserData.php

```diff
@@ -116,6 +116,7 @@ protected function getUserData(string $userId, bool $includeScopes = false): ?ar
 
 		// Find the data
 		$data['id'] = $targetUserObject->getUID();
+		$data['firstLogin'] = $targetUserObject->getFirstLogin() * 1000;
 		$data['lastLogin'] = $targetUserObject->getLastLogin() * 1000;
```

Comment:
this is totally not what I expect from an API when I receive a timestamp int. I would expect it to be unix timestamp, not javascript-based milliseconds …
While I see that you just add the firstLogin and follow the pattern, it hardens the problem and previously faulty implementation.

Maybe we can:
```suggestion
		$data['firstLoginTimestamp'] = $targetUserObject->getFirstLogin();
		$data['lastLoginTimestamp'] = $targetUserObject->getLastLogin();
		$data['lastLogin'] = $targetUserObject->getLastLogin() * 1000;
```

And deprecate `lastLogin` ?

Reply:
Agreed, though I would prefer ISO 8601 over unix timestamp

Reply:
+1 for ISO